tools/gfastats/macros.xml

```diff
@@ -1,5 +1,5 @@
 <macros>
-    <token name="@TOOL_VERSION@">1.3.0</token>
+    <token name="@TOOL_VERSION@">1.3.3</token>
```

Comment:
we the conda package is not yet available

Reply:
derp, sorry about that... why is bioconda trying to get a version that is not out yet, though: https://github.com/bioconda/bioconda-recipes/pull/36290

Reply:
I think Giulio has played with the builds in the last days.

Reply:
ah nevermind, it is a tag, I just didn't see it in the main releases tab, my bad: https://github.com/vgl-hub/gfastats/releases/tag/v1.3.4